### DISCUSSION TITLE
Set number of elements  of faker

### DISCUSSION THREAD

**Discussion by edisonfv** [TiagoSilvaPereira/vemto-issues#150] [Ideas & Suggestions]
Many times I need only 2 elements of identity, It would be good set the number faker element.

**Comment by TiagoSilvaPereira** [comment]
Hi @edisonfv sorry, I did not understand... could you please share more details?

**Comment by snappyio** [comment]
Sometimes Vemto default creates too much dummy date when seeding the database.

I often have to go and delete a bunch of records after the initial seed.

It would be good if we could define on a model/table how many records are created or even an option for no records.
